syft/src/main/java/org/openmined/syft/datasource/JobLocalDataSource.kt

```diff
@@ -114,4 +114,14 @@ internal class JobLocalDataSource {
         }
         return file.absolutePath
     }
+
+    @ExperimentalUnsignedTypes
+    fun getModelsPath(config: SyftConfiguration, jobId: String) = "${config.filesDir}/$jobId/models"
```

Comment:
the syft repository gets the config while creation. Pass it on to the job model and store it there. So you won't need to pass syft config anymore in these methods. THen finally remove config operand from the method calls in data downloaders